lib/server/bootevent.js

```diff
@@ -6,6 +6,34 @@ var UPDATE_THROTTLE = 1000;
 
 function boot (env, language) {
 
+  //////////////////////////////////////////////////
+  // Check Node version. Latest Node 8 LTS and Latest Node 10 LTS are recommended and supported.
+  // < 8        does not work, not supported
+  // >= 8.14.0  works, supported and recommended
```

Comment:
I believe this comment should now say `>= 8.15.0`

Reply:
fixed